cloudigrade/account/tasks.py

```diff
@@ -85,4 +87,52 @@ def enqueue_ready_volume(ami_id, volume_id, region):
     aws.check_volume_state(volume)
     messages = [{'ami_id': ami_id, 'volume_id': volume_id}]
 
-    add_messages_to_queue('ready_volumes', messages, 'ami_id')
+    add_messages_to_queue('ready_volumes', messages)
+
+
+@shared_task
+@rewrap_aws_errors
+def scale_up_inspection_cluster():
+    """
+    Scale up the "houndigrade" inspection cluster.
+
+    Returns:
+        None: Run as a scheduled Celery task.
+
+    """
+    if not aws.is_scaled_down(settings.HOUNDIGRADE_AWS_AUTOSCALING_GROUP_NAME):
+        # Quietly exit and let a future run check the scaling.
+        return
+
+    messages = read_messages_from_queue(
+        'ready_volumes',
+        settings.HOUNDIGRADE_AWS_VOLUME_BATCH_SIZE
+    )
+
+    if len(messages) == 0:
+        # Quietly exit and let a future run check for messages.
```

Comment:
Not sure if this is the right time, but I'd love to see some debug logging happen in situations like this.

Reply:
I'd considered about the same thing when I wrote this. I think I'll add some in now.

Reply:
info logs added in the latest commits. I figure start louder at info and we can lower it to debug when we decide it's too noisy.